src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocCorrect.java

```diff
@@ -196,4 +196,120 @@ void foo6() {}
          /** Stop instances being created. **/
          String twoSentences1() {return "Sentence one. Sentence two.";}
     };
+
+    class InputInlineSummaryJavadocParagraphCheck  {
```

Comment:
please move your tests to separate file to keep test Input reasonably sized.

Reply:
Done

Reply:
Done